src/main/java/example/CourierClientApp.java

```diff
@@ -53,6 +53,46 @@ public static void main(String[] args) throws IOException {
         SendEnhancedResponseBody sendEnhancedResponseBody1 = new SendService().sendEnhancedMessage(sendEnhancedRequestBody);
         System.out.println(sendEnhancedResponseBody1);
 
+        // Enhanced Send - simple object
+        Gson gson = new Gson();
+        SendEnhancedRequestBody sendEnhancedRequestBody = new SendEnhancedRequestBody();
+        SendRequestMessage sendRequestMessage = new SendRequestMessage();
+        HashMap<String, Object> toMap = gson.fromJson("{'email':'tejas@courier.com'}", HashMap.class);
+        
+        sendRequestMessage.setTo(toMap);
+        sendRequestMessage.setTemplate("my-template");
+        sendRequestMessage.setBrand_id("my-brand");
+        sendEnhancedRequestBody.setMessage(sendRequestMessage);
+
+        SendEnhancedResponseBody sendEnhancedResponseBody2 = new SendService().sendEnhancedMessage(sendEnhancedRequestBody);
+        System.out.println(sendEnhancedResponseBody2);
+
+        // Enhanced Send - list id, list pattern an user
+        SendEnhancedRequestBody sendEnhancedRequestBody = new SendEnhancedRequestBody();
```

Comment:
sendEnhancedRequestBody -> sendEnhancedRequestBody2

Reply:
same for response bodies... basically just make sure whole file does not have red squiggles